developer-guides/rest-api/chat/unstarmessage/README.md

```diff
@@ -1,26 +1,29 @@
 # Unpins a Chat Message
-Removes the star on the chat message for the authenticated user.
+
+Unpins the chat message for the authenticated user.
 
 | URL | Requires Auth | HTTP Method |
-| :--- | :--- | :--- | :--- |
-| `/api/v1/chat.unStarMessage` | `yes` | `POST` |
+| :--- | :--- | :--- |
+| `/api/v1/chat.unPinMessage` | `yes` | `POST` |
```

Comment:
This file is named `unstarmessage`, so I think this file is meant to be about the unstaring

Reply:
Whoops my bad, that's a mistake on my part. I'll fix it now. Apologies